content/en/gpu_monitoring/setup.md

```diff
@@ -8,15 +8,15 @@ This page provides instructions on setting up Datadog's GPU Monitoring on your i
 
```

Comment:
Can we update the sentence above the Prerequisites sentence to include more context about what advanced eBPF metrics offer? Something along the lines of 

"This page provides instructions on setting up Datadog's GPU Monitoring on your infrastructure. Follow the configuration instructions that match your operating environment in the table below.

To get additional insights and advanced eBPF metrics, like GPU core utilization, you can optionally opt-in to  enabling System Probe with privileged mode."